site/src/mdx-components.tsx

```diff
@@ -116,21 +117,17 @@ const Heading = ({ level, component, children, id }: HeadingProps) => {
     children,
   );
 
-  return id ? (
-    <>
-      <Box
-        component="a"
-        display="block"
-        id={id}
-        style={{
-          visibility: 'hidden',
-          scrollMarginTop: 120,
```

Comment:
Are we ok with removing the scroll margin? It was kind of nice having heading anchors consistently scroll to the same spot.

Reply:
Unless my Chrome is busted, I cannot see this having an effect with or without it. Are you able to replicate this as a regression?

Reply:
In both firefox and chrome, navigating between `h2`s and `h3`s in the aside makes the different scroll margins obvious. Ultimately it's because `h3`s have `xlarge` top padding but `h2`s have `xxlarge` top padding (and `h1s` have none).

Reply:
Ah need to hoist that to the element with the id. Also aligning it to the header height as defined in the docs page style sheet.

Reply:
Hmm, your change didn't fix it 🤔 
Also, the anchor on the `h1` is gone now?